scripts/import_osm.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+import sys
+import osmnx as ox
+from ipygis import get_connection_url
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.schema import CreateSchema
+from geoalchemy2 import Geometry, WKTElement
+from typing import Dict
+
+# test simple import now, convert to module later
+sys.path.insert(0, "..")
+from models import OSMPoint
+from osm_tags import tags_to_filter
+
+
+class OsmImporter(object):
+    """Import OSM data to PostGIS using Overpass API."""
+
+    def __init__(self, args: Dict):
+        args = self._parse_args(args)
+        self.slug = args["slug"]
+        self.bbox = args["bbox"]
+        self.city = args["city_name"]
+        self._engine = self._create_engine()
+
+    @staticmethod
+    def _parse_args(args) -> Dict:
+        """Parse input arguments to a dict."""
+        slug = args["city"]
+        if not slug:
+            raise AssertionError("You must specify the city name.")
+        # BBOX (minx, miny, maxx, maxy) takes precedence over city
+        arg_bbox = args["bbox"]
+        if arg_bbox:
+            minx, miny, maxx, maxy = map(float, arg_bbox.split(", "))
+            city_name = ""
+        else:
+            # support old city parameter too, if we want to let osmnx do geocoding
+            if slug == "COP":
+                city_name = "Copenhagen, Denmark"
+                minx, miny, maxx, maxy = 12.42000, 55.61000, 12.65000, 55.78000
+            elif slug == "HEL":
+                city_name = "Helsinki, Finland"
+                minx, miny, maxx, maxy = 24.82345, 60.14084, 25.06404, 60.29496
+            elif slug == "WAR":
+                city_name = "Warsaw, Poland"
+                minx, miny, maxx, maxy = 20.79057, 52.09901, 21.31300, 52.38502
+            else:
+                raise AssertionError("You must specify a bounding box or a known city.")
```

Comment:
Actually here we should always require boundingbox. The import.py script does geocoding for city names, so I don't think any importers should care about city names anymore, unless they specifically *need* a city name for some reason.